.github/util/initialize/action.yml

```diff
@@ -17,20 +17,48 @@ runs:
       with:
         node-version: "${{ inputs.node-version }}"
 
+    # See: https://github.com/dart-lang/sdk/issues/52266
+    - run: Invoke-WebRequest https://pub.dev
+      if: runner.os == 'Windows'
+      shell: pwsh
+
+    # See: https://github.com/orgs/community/discussions/131594
+    # The composite action requires an explict shell, but bash is not available on windows-arm64 runner.
+    # For the following commands conditionally use bash or pwsh based on the runner.os:
     - run: dart pub get
+      if: runner.os != 'Windows'
       shell: bash
 
+    - run: dart pub get
+      if: runner.os == 'Windows'
+      shell: pwsh
+
     - run: npm install
+      if: runner.os != 'Windows'
       shell: bash
 
+    - run: npm install
+      if: runner.os == 'Windows'
+      shell: pwsh
+
     - uses: bufbuild/buf-setup-action@v1.30.0
       with: {github_token: "${{ inputs.github-token }}"}
 
     - name: Check out the language repo
+      if: github.event_name == 'pull_request'
```

Comment:
This is to avoid running the `clone-linked-repo` during release build, which depend on `bash` and does not work on windows-arm64.

Reply:
Document this in a comment.